server.js

```diff
@@ -35,22 +39,31 @@ require("./config/passport");
     })
   );
 // Middleware
-app.use(bodyParser.urlencoded({ extended: true }))
-app.use(logger('dev'));
+router.get("/", function (req, res) {
+  res.get("index", {
+    user: req.user,
+  });
+});
+
+module.exports = router; 
+
+// app.use(bodyParser.urlencoded({ extended: true }))
+// app.use(logger('dev'));
 app.use(express.static('public'));
 app.use('/', router);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
   app.use(passport.initialize());
   app.use(passport.session());
-
+  
+// workout log 
+app.use('/', excerRoute); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
app.use("/", excerRoute); 
```